exercises/practice/binary-search-tree/.meta/config.json

```diff
@@ -22,5 +22,5 @@
   },
   "blurb": "Insert and search for numbers in a binary tree.",
   "source": "Josh Cheek",
-  "source_url": "https://twitter.com/josh_cheek"
+  "source_url": "https://www.deviantart.com/joshcheek"
```

Comment:
The link checker keeps failing on this one 🤔 
```
 ### Errors in exercises/practice/binary-search-tree/.meta/config.json

* [403] [https://www.deviantart.com/joshcheek](https://www.deviantart.com/joshcheek) | Failed: Network error: Forbidden
```

Reply:
Yes, I need to open a PR in problem specs to remove it. Some websites just refuse to respond with 200 to a script, and DeviantArt is one of them 😞